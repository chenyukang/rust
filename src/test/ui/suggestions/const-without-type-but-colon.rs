fn main() {
    const a: = 123;
    //~^ ERROR missing type for `const` item
    const b = 345;
    //~^ ERROR missing type for `const` item

    const c: = "hello";
    //~^ ERROR missing type for `const` item
}
