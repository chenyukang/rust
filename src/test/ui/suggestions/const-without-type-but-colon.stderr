error: missing type for `const` item
  --> $DIR/const-without-type-but-colon.rs:2:11
   |
LL |     const a: = 123;
   |           ^ help: provide a type for the constant: `a: i32`

error: missing type for `const` item
  --> $DIR/const-without-type-but-colon.rs:4:11
   |
LL |     const b = 345;
   |           ^ help: provide a type for the constant: `b: i32`

error: missing type for `const` item
  --> $DIR/const-without-type-but-colon.rs:7:11
   |
LL |     const c: = "hello";
   |           ^ help: provide a type for the constant: `c: &str`

error: aborting due to 3 previous errors

